src/ui/actions/timeline.ts

```diff
@@ -511,6 +511,19 @@ export function setFocusRegion(focusRegion: {
   return { type: "set_trim_region", focusRegion };
 }
 
+export function setFocusAroundTime(time: number, maxTime: number): UIThunkAction {
+  return (dispatch, getState) => {
+    const state = getState();
+    const zoomRegion = getZoomRegion(state);
+
+    const duration = zoomRegion.endTime;
+    const startTime = Math.max(time - maxTime / 2, 0);
+    const endTime = Math.min(time + maxTime / 2, duration);
```

Comment:
Yeah, that was the intended behavior so that we try to keep wherever the user clicked as a midpoint, and try to capture up to focus up to 15s on either end. But you're right, say you clicked at 0s, then you'd default the focus to 0s-15s. 

Also in general I have more confidence in the backend being able to focus properly with a smaller duration so I preferred it default to a smaller rather than a larger range. The user can easily adjust it as a follow-up if needed.

Reply:
That's fair. I don't feel strongly about this. Just seems like we're being unnecessarily harsh/strict in this case :D